pkgs/development/compilers/gcc/9/default.nix

```diff
@@ -5,7 +5,7 @@
 , langGo ? false
 , profiledCompiler ? false
 , staticCompiler ? false
-, enableShared ? true
+, enableShared ? !stdenv.targetPlatform.isMusl
```

Comment:
this is an ugly fix for protobuf https://github.com/NixOS/nixpkgs/issues/76873

Reply:
@FRidh Perhaps we could fix it better by removing `.la` files, like some packages like SDL do which say that pkgconfig is much better at handling this than libtool (which I think is true):

https://github.com/NixOS/nixpkgs/blob/201d9b8916dd04d1c2eaa6c306a7a603d4859ae3/pkgs/development/libraries/SDL2/default.nix#L78-L94

But when I try this here for gcc9 (on `master`, not `staging-next`), the GCC build fails even before `postInstall` is run:

```
/build/build/./gcc/xgcc -B/build/build/./gcc/ -xc++ -nostdinc /dev/null -S -o /dev/null -fself-test=../../gcc-9.2.0/gcc/testsuite/selftests
/nix/store/n9acaakxahkv1q3av11l93p7rgd4xqsf-bootstrap-tools/bin/bash: /build/build/./gcc/xgcc: No such file or directory
../../gcc-9.2.0/gcc/cp/Make-lang.in:178: recipe for target 's-selftest-c++' failed
make[3]: *** [s-selftest-c++] Error 127
rm gcc.pod
make[3]: Leaving directory '/build/build/gcc'
```

Impurity?